include/rdm/param_data.h

```diff
@@ -0,0 +1,585 @@
+/******************************************************************************
+ * Copyright 2019 ETC Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *******************************************************************************
+ * This file is a part of RDM. For more information, go to:
+ * https://github.com/ETCLabs/RDM
+ ******************************************************************************/
+
+/*! \file rdm/param_data.h
+ *  \brief Structures and packing/unpacking functions for the parameter data of standard PIDs.
+ *  \author Christian Reese*/
+#ifndef _RDM_PARAM_DATA_H_
+#define _RDM_PARAM_DATA_H_
+
+#include "etcpal/bool.h"
+#include "etcpal/error.h"
+#include "etcpal/inet.h"
+#include "rdm/message.h"
+
```

Comment:
Add a doxygen comment to organize the functions in this header into a new module, e.g.:
```
/*! \defgroup param_data Parameter Data
 *  \ingroup rdm
 *
 *  [describe the purpose of this module...]
 *
 *  @{
 */
```

And then close the group at the end of the file:
```
/*! @} */
```

Reply:
Done - wrote a \brief description for it.